python/lsst/daf/butler/core/schema.py

```diff
@@ -20,43 +20,18 @@
 # along with this program.  If not, see <http://www.gnu.org/licenses/>.
 
 from .utils import iterable
-from .config import Config
```

Comment:
Why did this move down? If you are grouping by local versus external imports shouldn't the `.utils` import also move?